docs/presentation/Content/CalculatedPropertiesSpecification.md

```diff
@@ -1,23 +1,49 @@
-| ---------- | --------- | -------------------------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `label`    | Yes       | `string`                                     |         | Label of the calculated property. Supports [localization](../Advanced/Localization.md).                                                                                                                                |
-| `value`    | Yes       | [ECExpression](../Advanced/ECExpressions.md) |         | Expression to calculate the value. The expression can use [ECInstance](../Advanced/ECExpressions.md#ecinstance) and [Ruleset Variables](../Advanced/ECExpressions.md#ruleset-variables-user-settings) symbol contexts. |
-| `priority` | No        | `number`                                     | `1000`  | Priority of the property. Determines the position of this property in UI components - higher priority means the property should be more visible.                                                                       |
+| Name                              | Required? | Type                                         | Default |
+| --------------------------------- | --------- | -------------------------------------------- | ------- |
+| [`label`](#attribute-label)       | Yes       | `string`                                     |         |
+| [`value`](#attribute-value)       | Yes       | [ECExpression](../Advanced/ECExpressions.md) |         |
+| [`priority`](#attribute-priority) | No        | `number`                                     | `1000`  |
 
-## Example
+### Attribute: `label`
 
-```JSON
-{
-  "priority": 9999,
-  "label": "@MyApp:Volume@",
-  "value": "this.Width * this.Height * this.Depth"
-}
+Specifies label of the calculated property. Supports [localization](../Advanced/Localization.md).
+
+```ts
+[[include:Content.Customization.CalculatedPropertiesSpecification.Label.Ruleset]]
 ```
+
+![Example of using "label" attribute](./media/calculatedpropertiesspecification-with-label-attribute.png)
+
+### Attribute: `value`
+
+Defines an expression to calculate the value. The expression can use [ECInstance](../Advanced/ECExpressions.md#ecinstance)
+and [Ruleset Variables](../Advanced/ECExpressions.md#ruleset-variables-user-settings) symbol contexts.
+
+```ts
+[[include:Content.Customization.CalculatedPropertiesSpecification.Value.Ruleset]]
+```
+
+![Example of using "value" attribute](./media/calculatedpropertiesspecification-with-value-attribute.png)
+
+### Attribute: `priority`
+
+> **Default value:** `1000`
+
+Assign a custom [Field.priority]($presentation-common) to the property. It's up to the UI component to make sure that priority
+is respected - properties with higher priority should appear before or above properties with lower priority.
+
```

Comment:
Am I interpreting this right: instance properties are not ordered by `priority` in the content response? Is this the point you were trying to get across?

Reply:
To be exact, priority is assigned to a `Field` which is created for one or more properties. We only assign priority to them and don't do any kind of sorting on the backend - that should be done by the component displaying the results.